contracts/HypERC20.sol

```diff
@@ -41,15 +41,21 @@ contract HypERC20 is ERC20Upgradeable, TokenRouter {
     }
 
     // called in `TokenRouter.transferRemote` before `Mailbox.dispatch`
-    function _transferFromSender(uint256 _amount) internal override {
+    function _transferFromSender(uint256 _amount)
```

Comment:
Since these are critical, maybe add a natspec

Reply:
natspec is only valid for external/public functions but adding